### ISSUE TITLE
 Coordinates

### ISSUE THREAD

**Issue by minhnhatspk** [jgrancher/react-native-sketch#32]
How can i get an array of 
coordinates before saving to images ?

**Comment by jgrancher** [comment]
So far there's no way of doing that. Is [this PR](https://github.com/jgrancher/react-native-sketch/pull/27) close to what you're looking for? (ability to keep a reference of drawing lines and / or delete them) ?

**Comment by minhnhatspk** [comment]
@jgrancher  Could you update the link in `PR` that you mentioned above

**Comment by jgrancher** [comment]
@minhnhatspk Yep sorry, I was talking about this one: https://github.com/jgrancher/react-native-sketch/pull/27
